### PR TITLE
Update descriptions for PointsAnnotationType and LineType

### PR DESCRIPTION
**Pull request by jtbandes**
**Public-Facing Changes**
Better docs for PointsAnnotationType and LineType

**Description**
Closes FG-1377

### CODE REVIEW

File: internal/schemas.ts [hunk 1]
```diff
@@ -357,9 +357,13 @@ const LineType: FoxgloveEnumSchema = {
   protobufEnumName: "Type",
   description: "An enumeration indicating how input points should be interpreted to create lines",
   values: [
-    { value: 0, name: "LINE_STRIP", description: "0-1, 1-2, ..., (n-1)-n" },
-    { value: 1, name: "LINE_LOOP", description: "0-1, 1-2, ..., (n-1)-n, n-0" },
-    { value: 2, name: "LINE_LIST", description: "0-1, 2-3, 4-5, ..." },
+    {
+      value: 0,
+      name: "LINE_STRIP",
+      description: "Connected line segments: 0-1, 1-2, ..., (n-1)-n",
```
Comment (esthersweon) [inline internal/schemas.ts:363]: It's somewhat confusing that the `-` in `(n-1)` means something different from the `-` in the `-n` suffix, but not sure how to edit..

Comment (esthersweon) [reply internal/schemas.ts:363]: Maybe just ending with the `...`, like in the `LINE_LIST` description?

File: internal/schemas.ts [hunk 2]
```diff
@@ -357,9 +357,13 @@ const LineType: FoxgloveEnumSchema = {
...
+    },
+    { value: 1, name: "LINE_LOOP", description: "Closed polygon: 0-1, 1-2, ..., (n-1)-n, n-0" },
```
Comment (esthersweon) [inline internal/schemas.ts:365]: Maybe just remove the `(n-1)-n` one here?

**Review by esthersweon — APPROVED** [review]
Left some minor comments.

**Comment by jtbandes** [comment]
I do think the n-1 provides some helpful context even though the formatting is a bit confusing. I think I'll lean towards keeping it.

**Pull request merged**